src/plot.jl

```diff
@@ -569,7 +569,7 @@ function fplot(f::Function, limits, args...; kvs...)
             pargs = [arg]
         elseif typeof(arg) <: Integer
             fopts[:min_points] = arg
-        elseif typeof(arg) <: FloatingPoint
+@compat elseif typeof(arg) <: AbstractFloat
```

Comment:
I think this should be
```julia
elseif typeof(arg) <: @compat AbstractFloat
```
This is also the reason for the failure of the travis builds.